src/website/apps/event/signals.py

```
@@ -0,0 +1,41 @@
+
+logger = logging.getLogger(__name__)
+
+
+def _clear_event_caches(sender, **kwargs):
+    """
+    Clear all event-related cache entries.
+
+    Uses cache.clear() because LocMemCache does not support
+    prefix-based deletion.  This is safe because:
+    - The cache is in-memory and rebuilds on next request.
+    - The TTL is short (5–10 min) anyway.
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)